pyt/helper_visitors/call_visitor.py

```diff
@@ -0,0 +1,71 @@
+import ast
+import re
+from collections import defaultdict, namedtuple
+from itertools import count
+
+from ..core.ast_helper import get_call_names_as_string
+from .right_hand_side_visitor import RHSVisitor
+
+
+class CallVisitorResults(
+    namedtuple(
+        "CallVisitorResults",
+        ["args", "kwargs", "unknown_args", "unknown_kwargs"]
+    )
+):
+    __slots__ = ()
+
+    def all_results(self):
+        for x in self.args:
+            yield from x
+        for x in self.kwargs.values():
+            yield from x
+        yield from self.unknown_args
+        yield from self.unknown_kwargs
+
+
+class CallVisitor(ast.NodeVisitor):
+    def __init__(self, trigger_str):
+        self.unknown_arg_visitor = RHSVisitor()
+        self.unknown_kwarg_visitor = RHSVisitor()
+        self.argument_visitors = defaultdict(lambda: RHSVisitor())
+        self._trigger_str = trigger_str
+
+    def visit_Call(self, call_node):
+        func_name = get_call_names_as_string(call_node.func)
+        trigger_re = r"(^|\.){}$".format(self._trigger_str.replace('.', r'\.'))
```

Comment:
`re.escape` might be more pythonic than `.replace('.', r'\.')`

Reply:
@KevinHock Thanks I didn't know about that function! :grinning: